src/historywork/GetRemoteFileWork.cpp

```diff
@@ -37,13 +41,23 @@ GetRemoteFileWork::getCommand()
     releaseAssert(mCurrentArchive);
     releaseAssert(mCurrentArchive->hasGetCmd());
     auto cmdLine = mCurrentArchive->getFileCmd(mRemote, mLocal);
-
+    // getCommand is called from RunCommandWork::onRun immediately before
+    // running the command.  Record the time here so that we can calculate
+    // throughput in onSuccess.
+    mDownloadStart = std::chrono::steady_clock::now();
+    if (getRetryCount() > 0)
+    {
+        mRetryRemoteFile.inc();
+    }
+    CLOG_INFO(History, "Downloading file '{}' from archive '{}', COMMAND='{}'",
```

Comment:
this can get pretty noisy if we're running a large catchup work. We try to keep the info log level tidy, and as compact as possible (verbose logs can cause log partitions to run out of space before they get rotated). We already have some DEBUG logs in `GetAndUnzipRemoteFileWork`, so perhaps that should be sufficient?